pull_request_template.md

```diff
@@ -2,11 +2,10 @@
 
 ## What does this PR do?
 Fixes #<issue_number>
-<!-- Please link the issue you're trying to fix with this PR, if none then please create an issue first. -->
 
 ## PR checklist
 Please check if your PR fulfills the following requirements:
-- [ ] Does this PR fix an existing issue?
+- [ ] Does this PR fix an existing issue or do you have well-explained what you changed in the PR description?
```

Comment:
Accordingly to Grammarly :D 

```suggestion
- [ ] Does this PR fix an existing issue, or do you have a well-explained what you changed in the PR description?
```

Reply:
I think the right thing would be to say:  Does this PR fix an existing issue, or do you have **well explained** what you changed in the PR description? (without the dash)

Reply:
`well` is super subjective, a little bit afraid by this

Reply:
What would you read instead @ferdi05?
What I want is details, so maybe simple "have you detailed"

Reply:
If not, then we could say that any PR submitted without an existing issue would fall in the category of "PRs that don't need an issue", and think of another way to say this to contributors.

Reply:
> I'm also wondering, do you think that some contributors would submit a PR which has a significant impact without creating an issue first?

It sometimes happened, despite the rules on the CONTRIBUTING.md file
In this case we discuss the "issue" on the PR, which is not really convenient, and depending on the repo, it often lead to open an issue

Reply:
What about: Have you listed the changes applied in the PR description (and why they are needed)?

Reply:
Like this one!

Reply:
WDYT?